src/android/CMakeLists.txt

```diff
@@ -79,9 +106,10 @@ endif()
 
 # Setup fbjni library
 find_library(FBJNI_LIBRARY fbjni PATHS ${FBJNI_LIB_DIR} NO_CMAKE_FIND_ROOT_PATH)
-target_include_directories(realm-js-android PUBLIC 
+target_include_directories(realm-js-android PUBLIC
     ${FBJNI_INCLUDE_DIR}
     ../../node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/turbomodule/core/jni/
```

Comment:
Based on change on line 14 should this line be removed?

Reply:
I'm not so certain.  Line 14 points to a specific file and this includes a directory.  Although, I could be wrong, as I'm not CMake experienced

Reply:
It does appear to build without.  I'll remove it and see if the tests continue to pass

Reply:
Removing those lines breaks the build.